src/AlgebraicGeometry/Schemes/AffineSchemes/Objects/Methods.jl

```diff
@@ -121,7 +121,7 @@ end
 @doc raw"""
     is_non_zero_divisor(f::RingElem, X::AbsAffineScheme)
 
-Checks if a ring element is a non-zero divisor
+Check if a ring element is a non-zero divisor
```

Comment:
```suggestion
Check if a ring element is a non-zero-divisor
```
This checks whether an element is not a zero-divisor (often also written as zerodivisor). Common notations in literature are non-zero-divisor or non-zerodivisor or nonzerodivisor (the latter is used in our primary source, the StacksProjekt, the first one is closest to the use of hyphenation in the Oscar documentation). The hyphenation 'non-zero divisor' is mathematically misleading, as it suggests a divisor which is non-zero.